crates/text/src/text.rs

```diff
@@ -1966,6 +1966,7 @@ impl BufferSnapshot {
                 .item()
                 .filter(|insertion| insertion.timestamp == anchor.timestamp)
             else {
+                self.insertions.items(&());
```

Comment:
This must have had a `dbg!` on it before a clippy run. I'm assuming we can/should delete this line.